to_do/to-do-app.js

```diff
@@ -0,0 +1,267 @@
+window.addEventListener("DOMContentLoaded", init);
+
+//Initialization which calls all the other functions
+function init() {
+  const toDoContainer = document.getElementById("list-container");
+  const addButtonNotStarted = document.querySelector(
+    ".new-task-button-not-started"
+  );
+  const addButtonInProgress = document.querySelector(
+    ".new-task-button-in-progress"
+  );
+
+  addButtonNotStarted.addEventListener("click", () =>
+    addTask(toDoContainer, false, 1)
+  );
+  console.log("clicked addbutton not started");
+  addButtonInProgress.addEventListener("click", () =>
+    addTask(toDoContainer, false, 2)
+  );
+  console.log("clicked addbutton in progress");
+
+  // Add saved tasks for Not Started and In Progress from localStorage
+  addTask(toDoContainer, true);
+
+  updateCount();
+}
+
+function addTask(toDoContainer, existing, category) {
+  console.log("add task");
+  const notStartedList = getNotStarted();
+  const inProgressList = getInProgress();
+  let toDoID = 0;
+
+  const date = new Date();
+  let day = date.getDate();
+  let month = date.getMonth() + 1;
+  let year = date.getFullYear();
+
+  let currentDate = `${month}-${day}-${year}`;
+
+  const toDoTemplate = {
+    id: Math.floor(Math.random() * 2000000),
+    title: "Default Title",
+    dueDate: `${currentDate}`,
+    label: "self",
+  };
+
+  if (existing) {
+    console.log("existing");

```

Comment:
if we are logging stuff in console, I would make it more detailed. Like existing what? + more context